zomes/holo_acorn/code/src/lib.rs

```diff
@@ -213,8 +213,26 @@ mod holo_acorn {
             validation_package: || {
                 hdk::ValidationPackageDefinition::Entry
             },
-            validation: | _validation_data: hdk::EntryValidationData<Profile>| {
-                Ok(())
+            validation: | validation_data: hdk::EntryValidationData<Profile>| {
+                match validation_data{
+                    hdk::EntryValidationData::Create{entry,..}=>{
+                        if entry.address!=AGENT_ADDRESS.to_string() {
```

Comment:
This is ROUGH, so this suggestion will need testing
```suggestion
                        let agent_address = validation_data.package.sources()[0]
                        if entry.address!=agent_address.to_string() {
```

https://docs.rs/hdk/0.0.40-alpha1/hdk/struct.ValidationData.html#method.sources

This will need to be changed for all of them. 
The reason for this, is that the validation callback can be run on ANY AGENT who receives the Entry via gossip... so you need to check the address of the Author agent, not the current agent